src/components/ViewRequest/Actions/SendRequest.tsx

```diff
@@ -107,7 +107,7 @@ const SendRequest = () => {
     const newData = {
       requestId,
       newStage: "PackageApproval",
-      newSubStage: "PackageApproval",
+      newSubStage: "SelectionPackageOSFApproval",
       eventTitle:
         "Forward Stage Change: Draft Package (HRL) to Package Approval",
```

Comment:
This re-enforces my question about having a different update handler for this particular stage, and not using the same calculate next stage function -- now the eventTitle specified here doesn't match the STAGES definition